docs/technical-setup/environment-setup/setup-conda.md

```diff
@@ -66,30 +67,67 @@ These commands will set the [recommended channels](https://docs.conda.io/project
 
-    - The [`awscli` package](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html) will allow you to interact with [data stored in the Amazon Web Services (AWS) S3 bucket](../../software-platforms/aws/index.md)
-    - The [`conda-lock` package](https://conda.github.io/conda-lock/) is used to create fully reproducible, cross-platform [conda environments for analysis modules](../../contributing-to-analyses/determining-requirements/determining-software-requirements.md#managing-software-dependencies-with-conda)
-    - The [`jq` package](https://jqlang.github.io/jq/) provides JSON parsing capabilities
-    - The [`pre-commit` package](https://pre-commit.com) will allow you to use [pre-commit hooks when contributing to analysis modules](../../contributing-to-analyses/working-with-git/making-commits.md#pre-commit-checks)
+    You will see a number of status messages flow by as packages are downloaded and installed.
+    When the process is complete, you'll see the following message in the terminal, which includes the commands to activate and deactivate the environment:
+
+    ```{ .console .no-copy title="Output message after conda environment install"}
+    Preparing transaction: done
+    Verifying transaction: done
+    Executing transaction: done
+    # To activate this environment, use
+    #
+    #     $ conda activate openscpca
+    #
+    # To deactivate an active environment, use
+    #
+    #     $ conda deactivate
+    ```
+
+    ??? info "Prefer to modify your base environment instead?"
+        While we recommended creating an `openscpca` environment, this is not strictly necessary.
+        If you prefer to add these packages to your base environment, use this command instead:
+        ```sh
+        conda env update -n base -f environment.yml
+        ```
+        If you experience package conflicts when running this command, we recommend you go back to setting up a dedicated `openscpca` environment.
+
 
-    !!! note
-        You may be prompted to enter **`y`** or **`n`** (yes or no) during this setup.
-        If/when this prompt appears, you should hit **`y`** to give conda permissions to proceed.
 
+1. As indicated in the message, activate the environment with the following command:
 
-1. At the end of the installation, you should see these messages in the terminal which indicate successful installation:
+    ```sh
+    conda activate openscpca
+    ```
 
+    At this point, your terminal prompt will show the prefix `(openscpca)`, which lets you know that you are in that conda environment.
+    In general, this is how conda indicates which environment, if any, the terminal is working in.
     <figure markdown="span">
-        ![Conda completed install output.](../../img/conda-success.png){width="275"}
+        ![Activate the openscpca environment](../../img/conda-activate-openscpca.png){width="300"}
     </figure>
```

Comment:
Yeah, I have a feeling you don't need this. If you really wanted something, I might do the following chunk of code (basically reproducing your image) But I actually think the text you have is probably enough, especially given the potential variety in bash prompts.
````
```{ .console .no-copy title="Activated conda environment"}
bash$ conda activate openscpca
(openscpca) bash$ █
```
````

Reply:
> But I actually think the text you have is probably enough, especially given the potential variety in bash prompts.